### PR TITLE
Fix sporadic failure of seahorse.pm

### PR DESCRIPTION
**Pull request by Zaoliang**
- increase timeout for assert_click to 60 sec
- see poo#33625 for more details
- verification run:
  http://e13.suse.de/tests/710#step/seahorse/9

### CODE REVIEW

File: tests/x11/seahorse.pm [hunk 1]
```diff
@@ -32,9 +32,9 @@ sub run {
     send_key "ret";                                   # Next field (confirm PW)
     type_password;                                    # Re-type user password
     send_key "ret";                                   # Confirm password
-    assert_and_click "seahorse-default_keyring", 'right';    # right click the new keyring
-    assert_and_click "seahorse-set_as_default";              # Set the new keyring as default
-    send_key "alt-f4";                                       # Close seahorse
+    assert_and_click "seahorse-default_keyring", 'right', 60;    # right click the new keyring
```
Comment (rwx788) [inline tests/x11/seahorse.pm]: How sure we are that it helps? How many runs have you performed?

Comment (Zaoliang) [reply tests/x11/seahorse.pm]: over 20 times

File: tests/x11/seahorse.pm [hunk 2]
```diff
@@ -32,9 +32,9 @@ sub run {
...
+    assert_and_click "seahorse-set_as_default";                  # Set the new keyring as default
```
Comment (okurz) [inline tests/x11/seahorse.pm]: In the original test failure as linked in the progress ticket this line fails with timeout, see https://openqa.opensuse.org/tests/639513/file/autoinst-log.txt so why did you bump the timeout on the previous call?

**Comment by Zaoliang** [comment]
@okurz yes, you're right. I did checkout the changes and add this at right line :) Now it got correct, thanks.

**Comment by nilxam** [comment]
I suppose this change breaks https://openqa.opensuse.org/tests/644696#step/seahorse/11 , the second parameter is `button click` not the timeout.

*<<< testapi::assert_and_click(mustmatch='seahorse-set_as_default', button=60, timeout=30)*

**Comment by Zaoliang** [comment]
@nilxam yes, thanks. Will correct this.

**Pull request merged**